### PR TITLE
Lazy badges

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull-request makes the badges resolution lazy allowing extensions to provides badges.

It removed the gouvfr badges as they can be provided by gouvfr itself.

### CODE REVIEW

**Comment by davidbgk** [comment]
Tests are failing.

**Comment by noirbizarre** [comment]
Test are passing now with the refatoring.

File: udata/core/badges/models.py [hunk 1]
```diff
@@ -13,7 +13,40 @@
...
+            raise db.ValidationError('Unknown badge type: %s' % self.kind)
+        return super(Badge, self).validate(clean=clean)
+
+
+class BadgesList(db.EmbeddedDocumentListField):
+    def __init__(self, *args, **kwargs):
+        return super(BadgesList, self).__init__(Badge, *args, **kwargs)
+
+    def validate(self, value):
+        print 'validate badges', value, len(value)
```
Comment (davidbgk) [inline udata/core/badges/models.py]: To remove.

File: udata/core/reuse/models.py [hunk 2]
```diff
@@ -79,7 +64,7 @@ class Reuse(db.Datetimed, WithMetrics, BadgeMixin, db.Document):
     datasets = db.ListField(
         db.ReferenceField('Dataset', reverse_delete_rule=db.PULL))
     tags = db.ListField(db.StringField())
-    badges = db.ListField(db.EmbeddedDocumentField(ReuseBadge))
+    # badges = db.ListField(db.EmbeddedDocumentField(ReuseBadge))
```
Comment (davidbgk) [inline udata/core/reuse/models.py]: To remove.

**Comment by davidbgk** [comment]
Should we generalize that approach to discussions/issues?

**Pull request merged**